apps/allocations/arapp.json

```diff
@@ -22,15 +22,21 @@
       "network": "rpc",
       "appName": "allocations.aragonpm.eth"
     },
-    "rinkeby": {
+    "staging": {
       "registry": "0x98df287b6c145399aaa709692c8d308357bc085d",
-      "appName": "tps-allocations.open.aragonpm.eth",
+      "appName": "allocations-staging.open.aragonpm.eth",
       "wsRPC": "wss://rinkeby.eth.aragon.network/ws",
       "network": "rinkeby"
     },
-    "staging": {
+    "continuous-deployment": {
       "registry": "0x98df287b6c145399aaa709692c8d308357bc085d",
-      "appName": "allocations-staging.open.aragonpm.eth",
+      "appName": "alllocations-cd.open.aragonpm.eth",
+      "wsRPC": "wss://rinkeby.eth.aragon.network/ws",
+      "network": "rinkeby"
+    },
+    "rinkeby": {
```

Comment:
The order of these mismatches the order in Address Book's `arapp`. I'd prefer we keep them consistent.